src/cell_hydro.h

```diff
@@ -61,6 +61,24 @@ struct cell_hydro {
     /*! Linked list of the tasks computing this cell's hydro density. */
     struct link *density;
 
+    /*! Linked list of the tasks computing this cell's hydro density pack. A. Nasar */
+    struct link *density_pack;
+    struct link *density_unpack;
+    /*! Linked list of the tasks computing this cell's hydro force pack. */
+    struct link *force_pack;
+    struct link *force_unpack;
+    /*! Linked list of the tasks computing this cell's hydro gradient pack. */
+    struct link *gradient_pack;
+    struct link *gradient_unpack;
+
+    struct task *d_pack;
```

Comment:
Since we just talked about and so that we don't forget: this pointer and the following five are likely unnecessary given the linked lists above.